React-Homeworks/src/components/FoodCard/FoodCard.css

```diff
@@ -0,0 +1,58 @@
+.card__img {
+    width: 100%;
+}
+
+.card__div-img {
+    width: 8rem;
+}
+
+.card__h2 {
+    justify-content: space-between;
+}
+
+.card {
+    /* width: 50%; */
+    /* width: 25rem; если надо респонсив меняющиеся колонки */
+    justify-content: center;
+    align-items: center;
+}
+
+.card__wrapper {
+    background-color: var(--color-white);
+    padding: 1rem;
+    gap: 1rem;
+    border-radius: 0.3rem;
+    border: 0.1rem solid rgba(53, 184, 190, 0.15);
+}
+
+#quantityInput {
+    -webkit-appearance: none;
```

Comment:
we don't need to use prefixes in our source code. There are many of additional tools to add it during the build process